pallets/proofs-dealer/src/types.rs

```diff
@@ -50,10 +106,29 @@ pub type ProvidersPalletFor<T> = <T as crate::Config>::ProvidersPallet;
 /// Syntactic sugar for the Provider type used in the proofs pallet.
 pub type ProviderFor<T> = <<T as crate::Config>::ProvidersPallet as ProvidersInterface>::Provider;
 
+/// Syntactic sugar for the ForestVerifier type used in the proofs pallet.
+pub type ForestVerifierFor<T> = <T as crate::Config>::ForestVerifier;
+
+/// Syntactic sugar for the ForestVerifier::Proof type used in the proofs pallet.
+pub type ForestVerifierProofFor<T> =
+    <<T as crate::Config>::ForestVerifier as CommitmentVerifier>::Proof;
+
+/// Syntactic sugar for the KeyVerifier type used in the proofs pallet.
+pub type KeyVerifierFor<T> = <T as crate::Config>::KeyVerifier;
+
+/// Syntactic sugar for the KeyVerifier::Proof type used in the proofs pallet.
+pub type KeyVerifierProofFor<T> = <<T as crate::Config>::KeyVerifier as CommitmentVerifier>::Proof;
+
 /// Syntactic sugar for the type of NativeBalance pallet.
 pub type BalancePalletFor<T> = <T as crate::Config>::NativeBalance;
 
 /// Syntactic sugar for the type of Balance used in the NativeBalances pallet.
 pub type BalanceFor<T> = <<T as crate::Config>::NativeBalance as fungible::Inspect<
     <T as frame_system::Config>::AccountId,
 >>::Balance;
+
+/// Syntactic sugar for the type of RandomnessProvider type used in the proofs pallet.
+pub type RandomnessProviderFor<T> = <T as crate::Config>::RandomnessProvider;
+
+/// Syntactic sugar for the Randomness Output type used in the proofs pallet.
+pub type RandomnessOutputFor<T> = <T as frame_system::Config>::Hash;
```

Comment:
I don't mind this, but I would make this be an alias over the `RandomnessProvider` to ensure the hash corresponds directly to what the provider should be outputting.

Reply:
I agree, but in this case, the `RandomnessProvider` is something that implements the `Randomness` trait and it doesn't have an associated type for it's output. The output is specified as a type parameter, and I do that [here](https://github.com/Moonsong-Labs/storage-hub/blob/1df731e9c35ce3c6fe4d1bcf115ae950356a66cb/pallets/proofs-dealer/src/lib.rs#L71), specifying that it should be a `Hash`.